.bumpversion.cfg

```diff
@@ -0,0 +1,7 @@
+[bumpversion]
+current_version = 0.0.1
```

Comment:
Why is this version different than the contracts version? Wouldn't it make sense to keep the package & contracts version in sync?

Reply:
a simplest answer would be that you might want to update python code (i.e. contract manager) without changing on-chain contract's version

Reply:
Ah, right. We can keep the `major` (maybe also `minor`) version synced. Thanks.